src/main/java/uk/gov/hmcts/ccd/domain/service/createevent/AuthorisedCreateEventOperation.java

```diff
@@ -255,6 +257,7 @@ private void verifyCaseFieldsAccess(Map<String, JsonNode> newData, CaseDetails e
             MAPPER.convertValue(existingCaseDetails.getData(), JsonNode.class),
             caseTypeDefinition.getCaseFieldDefinitions(),
             accessProfiles)) {
+            log.error("Error validating case field CRUD in case: {}",existingCaseDetails.getReference());
```

Comment:
we would like to log the field name, type, user roles and the current CRUD access etc too

Reply:
I noticed a log present in [AccessControlServiceImpl](https://github.com/hmcts/ccd-data-store-api/blob/a6f4384ab2842affe6f5e4efcecb9f685c0341f5/src/main/java/uk/gov/hmcts/ccd/domain/service/common/AccessControlServiceImpl.java#L377). Currently this is a debug log that throws the details you've suggested, besides the field type.

I'll go ahead and undo the changes in this file since the existing log covers it